tsconfig.json

```diff
@@ -8,7 +8,12 @@
     "inlineSourceMap": true,
     "declaration": true,
     "newLine": "LF",
-    "outDir": "dist"
+    "outDir": "dist",
+    "lib": [
```

Comment:
Do we get the dom lib though? I was getting type errors because tsc couldn't find HTMLElements and so forth.